### PR TITLE
Minor improvements to the head and neck example

### PR DESCRIPTION
**Pull request by sankhesh**
- Ability to switch between lung(L key) and bone (B key) transfer functions
- Performance tweaks during interaction

### CODE REVIEW

File: Examples/Python/cinematic_rendering/head_and_neck_front.py [hunk 1]
```diff
@@ -92,9 +97,37 @@
...
+volume_prop_l.SetColor(color_l)
+
 volume = vtk.vtkVolume()
 volume.SetProperty(volume_prop)
 
+def on_key_press(style, event):
+    if style.GetInteractor().GetKeySym() == 'l':
+        logging.info('Switching to lung TF')
+        volume.SetProperty(volume_prop_l)
+        renWin.Render()
```
Comment (psavery) [inline Examples/Python/cinematic_rendering/head_and_neck_front.py:125]: Nice! Thanks for adding a new set of transfer function parameters.

**Review by psavery — APPROVED** [review]
Looks good to me!

**Comment by psavery** [comment]
The CI is only failing because cibuildwheel is trying to build wheels for python3.12, and the current version of VTK we depend on does not have wheels yet for python3.12. We can merge anyways and fix that issue in a separate PR.

**Comment by sankhesh** [comment]
Thank you @psavery

**Pull request merged**